### PR TITLE
Add tests to 1.6.1.4

### PR DESCRIPTION
**Pull request by isuruuy429**
## Purpose
Add test and relevant artifacts to 1.6.1.4.

### CODE REVIEW

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/artifact.xml [hunk 1]
```diff
@@ -107,4 +107,8 @@
     <artifact name="1_2_1_API_pox_to_json_using_payload_factory" groupId="org.wso2.ei.product-scenarios.scenario_01.api" version="1.0.0" type="synapse/api" serverRole="EnterpriseServiceBus">
         <file>src/main/synapse-config/api/1_2_1_API_pox_to_json_using_payload_factory.xml</file>
     </artifact>
+    <artifact name="1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild" groupId="org.wso2.ei.product-scenarios.scenario_01.proxy-service" version="1.0.0" type="synapse/proxy-service" serverRole="EnterpriseServiceBus">
+        <file>src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml</file>
+>>>>>>> add tests to 1.6.1.4
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/artifact.xml]: Conflict remains

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<proxy name="1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <property name="TEST" scope="default">
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml]: Shall we provide meaning property name

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.1-Modify-payload-by-adding-a-child-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddChildToXMLTest.java [hunk 3]
```diff
@@ -89,6 +89,50 @@ public void testPassThroughProxyTemplate() throws IOException, XMLStreamExceptio
...
+        String expectedResponse =
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                        + "<soap:Envelope xmlns:soap=\"http://schemas.xmlsoap.org/soap/envelope/\">\n"
+                        + "   <soap:Body>\n"
+                        + "      <Company>\n"
+                        + "         <Name>WSO2</Name>\n"
+                        + "         <Location>Colombo, Sri Lanka</Location>\n"
+                        + "         <Dept>\n"
+                        + "            Engineering\n"
+                        + "            <Employee>\n"
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.1-Modify-payload-by-adding-a-child-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddChildToXMLTest.java]: We should embed the property content as a child, not append to child content

Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.1-Modify-payload-by-adding-a-child-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddChildToXMLTest.java]: This is done.

Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.1-Modify-payload-by-adding-a-child-using-enrich-mediator/src/test/java/org.wso2.carbon.ei.scenario.test/AddChildToXMLTest.java]: This is done

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml (hunk 2 above)
Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml]: Done

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/artifact.xml (hunk 1 above)
Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/artifact.xml]: Resolved

File: product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+<proxy name="1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <property name="employeeInfo" scope="default">
+                <employeeInfo xmlns="">
+                    <firstName>Jacque</firstName>
+                    <lastName>Kallis</lastName>
+                    <team>EI</team>
+                </employeeInfo>
```
Comment (milindaperera) [inline product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml]: Shall we make this element as follows:
<employeeInfo xmlns="">
  <employee>
      <firstName>Jacque</firstName>
      <lastName>Kallis</lastName>
      <team>EI</team>
  </employee>
</employeeInfo>

Comment (isuruuy429) [reply product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml]: This is done.

**Comment by isuruuy429** [comment]
Covers in https://github.com/wso2/product-ei/pull/3155

**Pull request closed without merging**